stage0/gc.go

```diff
@@ -125,9 +189,20 @@ func ForceMountGC(path, uuid string) error {
 		return fmt.Errorf("error getting mounts for pod %s from mountinfo: %v", uuid, err)
 	}
 
+	for i := len(mnts) - 1; i >= 0; i -= 1 {
+		mnt := mnts[i]
+		if mnt.mode != "-" && strings.Contains(mnt.mode, "shared") {
```

Comment:
In the way you parse each line of mountinfo, you cannot have a "-" in `mnt.mode`.